docs/gettingstarted/query_test_report.rst

```diff
@@ -167,6 +167,21 @@ filter test and report all **FAIL** tests.
 
 .. command-output:: buildtest report --filter tags=tutorials --format name,id,state --failure
 
+Find Tests by Start and End Dates(``buildtest report --start --end``)
+--------------------------------------------------------
+
+The ``buildtest report --start --end`` command can be used to retrieve all tests in [start, end] dates range.
```

Comment:
Say something like "The ``buildtest report --start`` and ``buildtest report --end``  command ..."